### PR TITLE
Update oauth-with-online-and-on-premises.md

### PR DESCRIPTION
**Pull request by msbemba**
added link to check compatibility

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ This topic applies to integration with Exchange Server 2013 through 2019.
 
 - For information about keyboard shortcuts that may apply to the procedures in this topic, see [Keyboard shortcuts in the Exchange admin center]( https://go.microsoft.com/fwlink/p/?LinkId=746512).
 
+- For Information on compatibility, see [Skype for Business compatibility with Office apps](Skype/SfbServer/plan-your-deployment/clients-and-devices/compatibility-with-office.md)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md]:
```suggestion
- For information about compatibility, see [Skype for Business compatibility with Office apps](https://docs.microsoft.com/en-us/skypeforbusiness/plan-your-deployment/clients-and-devices/compatibility-with-office).
```

File: Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ This topic applies to integration with Exchange Server 2013 through 2019.
 
 - For information about keyboard shortcuts that may apply to the procedures in this topic, see [Keyboard shortcuts in the Exchange admin center]( https://go.microsoft.com/fwlink/p/?LinkId=746512).
 
+- For information about compatibility, see [Skype for Business compatibility with Office apps](https://docs.microsoft.com/en-us/skypeforbusiness/plan-your-deployment/clients-and-devices/compatibility-with-office).
```
Comment (nenonix) [inline Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md]:
```suggestion
- For information about compatibility, see [Skype for Business compatibility with Office apps](https://docs.microsoft.com/skypeforbusiness/plan-your-deployment/clients-and-devices/compatibility-with-office).
```

**Comment by scanum** [comment]
@carolynblanding @LanaChin Changes look correct. Could you please review and merge? Thanks!

**Comment by scanum** [comment]
@LanaChin Changes look correct. Could you please review and merge? Thanks!

**Comment by LanaChin** [comment]
@shars - over to you.

**Comment by SharS** [comment]
Verified link.

**Pull request merged**